correios/models/posting.py

```diff
@@ -51,6 +51,9 @@
 MIN_CYLINDER_LENGTH, MAX_CYLINDER_LENGTH = 18, 105  # cm
 MIN_SIZE, MAX_SIZE = 29, 200  # cm
 MIN_CYLINDER_SIZE, MAX_CYLINDER_SIZE = 28, 200  # cm
+MAX_MECHANIZABLE_SIZE = 70  # cm
+NON_MECHANIZABLE_COST = Decimal('20.00')
```

Comment:
esse nome mechanizable é de alguma diretriz dos correios?

Reply:
Sim.
>Os Correios realizaram adequações nos critérios de cobrança de encomendas (SEDEX e PAC) para objetos chamados "não mecanizáveis".

Reply:
I think `MAX_MECHANIZABLE_SIZE` is not descriptable enough. What do you think about `MAX_MECHANIZABLE_OBJECTS_SIZE`? The same for cost variable.

Reply:
🤔 I think its ok `MAX_MECHANIZABLE_SIZE `.
I you look there is others similars constants: MIN_SIZE, MAX_SIZE, MIN_WIDTH, MAX_WIDTH...